package-lock.json

```diff
@@ -11,6 +11,7 @@
       "dependencies": {
         "@actions/core": "1.9.1",
         "@actions/github": "5.1.0",
+        "fast-glob": "3.2.12",
```

Comment:
I don't have a strong opinion about the library. The main alternative might be https://www.npmjs.com/package/glob